backend/src/email/models/email-message-tx.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { User } from '@core/models'
+
+// identical to CampaignS3ObjectInterface; to tweak and abstract if necessary
+interface AttachmentS3ObjectInterface {
+  key?: string
+  bucket?: string
+  filename?: string
+  temp_filename?: string
+  error?: string
+}
+
+export enum TransactionalEmailMessageStatus {
+  Unsent = 'UNSENT',
+  // AttachmentTooLargeError = 'ATTACHMENT_TOO_LARGE_ERROR', // unused
+  // TooManyAttachmentsError = 'TOO_MANY_ATTACHMENTS_ERROR', // unused
+  // InvalidParamsError = 'INVALID_PARAMS_ERROR', // unused; these are errors caught by Joi
+  InvalidFromAddressError = 'INVALID_FROM_ADDRESS_ERROR',
+  RateLimitError = 'RATE_LIMIT_ERROR',
+  InvalidMessageError = 'INVALID_MESSAGE_ERROR',
+  UnsupportedFileTypeError = 'UNSUPPORTED_FILE_TYPE_ERROR',
+  MaliciousFileError = 'MALICIOUS_FILE_ERROR',
+  BlacklistedRecipientError = 'BLACKLISTED_RECIPIENT_ERROR',
+  Accepted = 'ACCEPTED',
+  Sent = 'SENT',
+  // unsupported for now
+  // Bounced = 'BOUNCED',
+  // Delivered = 'DELIVERED',
+  // Opened = 'OPENED',
+  // Complaint = 'COMPLAINT',
+}
+
+@Table({ tableName: 'email_messages_tx', underscored: true, timestamps: true })
+export class EmailMessageTx extends Model<EmailMessageTx> {
+  @Column({
+    type: DataType.BIGINT,
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+  })
+  id!: number
+
+  @ForeignKey(() => User)
+  @Column({ type: DataType.STRING, allowNull: false })
+  userEmail!: string
```

Comment:
I was thinking:
- we should have the uniqueness requirement in `user.email` anyway
- by saving `userEmail` here, we can visually browse through the db using Tableplus to get a sense of who's sending what, rather than having to relate it back to the `users` table all the time.

Not sure if there are any downsides